src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Utilities/Converters.cs

```diff
@@ -99,7 +99,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
             foreach (object v in values)
             {
                 // Any false, the result is false
-                if (v is bool && !(bool)v)
+                if (v is bool b && !b)
```

Comment:
As above, `v is false` should work

Reply:
As above.

Reply:
This is a multi-value converter that boxes every input and the output.

Take a look at c494b64944a0300cb0ee6271eb6e95ec34ba47ef which prevents any boxing allocations other than two during the type initialiser. Does that reduce readability too much?